modules/era/sql/era_mob_droplist.sql

```diff
@@ -26387,6 +26386,11 @@ INSERT INTO `mob_droplist` VALUES (3233,0,0,1000,1875,@ALWAYS); -- Ancient Beast
 INSERT INTO `mob_droplist` VALUES (3233,0,0,1000,1875,@VCOMMON); -- Ancient Beastcoin (Very Common, 24%)
 INSERT INTO `mob_droplist` VALUES (3233,0,0,1000,1875,@VCOMMON); -- Ancient Beastcoin (Very Common, 24%)
 
+-- ZoneID: 173 - Jelly
+INSERT INTO `mob_droplist` VALUES (3234,0,0,1000,637,430); -- Vial Of Slime Oil (43.0%)
+INSERT INTO `mob_droplist` VALUES (3234,0,0,1000,637,220); -- Vial Of Slime Oil (22.0%)
+INSERT INTO `mob_droplist` VALUES (3234,0,0,1000,637,140); -- Vial Of Slime Oil (14.0%)
```

Comment:
All of our drop rates on ASB must follow the provided chart SE gave some years ago. You'll have to use the categories like you can see used on other entires in the droplist SQL. You can find the chart here: https://ffxiclopedia.fandom.com/wiki/Treasure_Hunter under the March 2020 Developer Comments

Reply:
I see. I dont think its possible to get the correct drop rate using only the categories. Though looking at this again, i dont think the drop rates in this PR are correct anyway. 

Setting each drop to `COMMON` would make the chance of getting at least one slime oil roughly 38%, which is close to the ffxidb value. Would that be sufficient?

Reply:
The problem is there isn't any reported instances of jelly dropping more than 1 slime oil. let alone 3.
http://ffxidb.com/zones/173/jelly

Reply:
I guess the two sources have conflicting data.

Reply:
I don't understand. There was already 3 drops in the code before this change. Also, FFXIClopedia also suggests 3 drops are possible: https://ffxiclopedia.fandom.com/wiki/Jelly

Farthest back i've found in the FFXIClopedia history on a mention of 3 drops is 2008: https://ffxiclopedia.fandom.com/wiki/Jelly?oldid=768432

Reply:
@CriticalXI is it okay to set all 3 drops to `COMMON`? That would make it a ~38% chance to drop at leaast one.

Reply:
Sure, and we've actually got a droplist with that setup already. The ID is 567.